mainnet-1/specs/axelar.json

```diff
@@ -0,0 +1,2656 @@
+{
+    "proposal": {
+        "title": "Add Specs: Axelar",
+        "description": "Adding new specification support for relaying axelar data on Lava",
+        "specs": [
+            {
+                "index": "AXELAR",
+                "name": "axelar mainnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDK50",
```

Comment:
Axelar uses Cosmos SDK v0.45. Perhaps we could use the Cosmos SDK spec with v0.47 already on chain if there are no API differences?

Reply:
Updated the import parameter. Replaced the COSMOSSDK50 specification with COSMOSSDK